bool thruster_status
float32 linear_sense
float32 angular_sense
float32 vertical_sense
int32 camera
